src/http/server/response.cr

```diff
@@ -40,6 +40,30 @@ class HTTP::Server
       output.response = self
     end
 
+    def remote_address
+      case @io
+      when IPSocket
+        io = @io
+      when UNIXSocket
```

Comment:
You can merge it with the above branch using `when IPSocket, UNIXSocket`. ditto below.